File: src/shared/Foundation.Template.Domain/Models/Dispositions/OrganisationTypeDisposition.cs
using System;

namespace Foundation.Template.Domain.Models
{
    public class OrganisationTypeDisposition
    {
        public Guid Id { get; set; }
        public Guid ColumnId { get; set; }
        public int Index { get; set; }
        public bool Hidden { get; set; }
        public bool Disabled { get; set; }
    }
}

File: src/app/Foundation.Extension.Admin-UI/domain/models/dispositions/organisationTypeDisposition.ts
export class OrganisationTypeDisposition {
    id: string;
    columnId: string;
    index: number;
    hidden: boolean;
    disabled: boolean;

    constructor(params: OrganisationTypeDispositionDTO) {
        this.id = params.id;
        this.columnId = params.columnId;
        this.index = params.index;
        this.hidden = params.hidden;
        this.disabled = params.disabled;
    }
}

export interface OrganisationTypeDispositionDTO {
    id: string;
    columnId: string;
    index: number;
    hidden: boolean;
    disabled: boolean;
}

export interface UpdateOrganisationTypeDispositionDTO {
    columnId: string;
    disabled: boolean;
    hidden: boolean;
    index: number;
}